Blob.js

```diff
@@ -256,7 +256,7 @@
 
 		function FakeBlobBuilder () {
 			function isDataView (obj) {
-				return obj && Object.prototype.isPrototypeOf.call(DataView, obj);
+				return obj && Object.prototype.isPrototypeOf(DataView, obj);
```

Comment:
This now tests if the DataView class is an Object and always returns true.

The current version is also broken and always returns false.

This should work correctly:
```suggestion
				return obj && Object.prototype.isPrototypeOf.call(DataView.prototype, obj);
```

Reply:
Thanks for this, change made.  I'm a little out of my depth here. :)